nixos/modules/services/torrent/deluge.nix

```diff
@@ -20,9 +60,93 @@ in {
             Number of files to allow deluged to open.
           '';
         };
+
+        coreConf = mkOption {
+          type = types.attrs;
+          example = {
+            download_location = "/srv/torrents/";
+            max_upload_speed = "1000.0";
+            share_ratio_limit = "2.0";
+            allow_remote = true;
+          };
+          description = ''
+            Deluge core configuration for the core.conf file. Only has
```

Comment:
Do you know of a reference we could link to for the available options? You can add `daemon_port` and `listen_ports` to the example to make users aware of these options.

Reply:
Unfortunately I haven't found any reference to the configuration options , though you can get the current values with the command `deluge-console "config"`. I have added `daemon_port` and `listen_ports` to the example.